.github/workflows/main.yml

```
@@ -134,10 +134,24 @@ jobs:
         run: |
           sudo apt update
           sudo apt install mysql-client
-          LASTEST_TAG=$(curl -s GET https://api.github.com/repos/datafuselabs/databend/tags\?per_page\=1 | jq -r '.[].name')
-          curl -LJO https://github.com/datafuselabs/databend/releases/download/${LASTEST_TAG}/databend-${LASTEST_TAG}-x86_64-unknown-linux-musl.tar.gz
-          mkdir ./databend && tar xzvf databend-${LASTEST_TAG}-x86_64-unknown-linux-musl.tar.gz -C ./databend
-          ./databend/bin/databend-query &
+          
+          # The first program: Download databend from github (unstable)
+          # LASTEST_TAG=$(curl -s GET https://api.github.com/repos/datafuselabs/databend/tags\?per_page\=1 | jq -r '.[].name')
+          # curl -LJO https://github.com/datafuselabs/databend/releases/download/${LASTEST_TAG}/databend-${LASTEST_TAG}-x86_64-unknown-linux-musl.tar.gz
+          # mkdir ./databend && tar xzvf databend-${LASTEST_TAG}-x86_64-unknown-linux-musl.tar.gz -C ./databend
+          # ./databend/bin/databend-query &
+          
+          # The second program: git clone and then compiled to install databend (The dependencies are not easy to solve, and the use time is more) 
+          # Dependecy protoc unresolved
+          # TODO: This method can be replaced with https://databend.rs/doc/contributing/building-from-source#build-with-docker-image
+          # git clone https://github.com/datafuselabs/databend.git
+          # cd databend && cargo build --bin databend-query --release
+          # ./target/release/databend-query &
+
+          # The third program: Download databend-query from docker hub
+          docker pull datafuselabs/databend-query
+          docker run --net=host datafuselabs/databend-query --mysql-handler-host 0.0.0.0 &
```

```suggestion
          docker run --net=host datafuselabs/databend-query
```

If we use `--net=host`, we don't need to specify the `mysql-handler-host` anymore.